test/bpf_tests/bpf_test.go

```diff
@@ -33,9 +34,17 @@ import (
 	"github.com/cilium/cilium/pkg/byteorder"
 	"github.com/cilium/cilium/pkg/datapath/link"
 	"github.com/cilium/cilium/pkg/monitor"
+	"github.com/cilium/coverbee"
+	"github.com/dylandreimerink/gocovmerge"
+	"golang.org/x/tools/cover"
```

Comment:
Nit: These should be in the second grouping as 3rd party imports

Reply:
Tip: `goimports -local github.com/cilium/cilium` will sort the imports correctly